concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -1977,12 +1977,11 @@ impl ConsensusContainer {
 
     pub fn send_finalization(&self, genesis_index: u32, msg: &[u8]) -> ConsensusFfiResponse {
         wrap_send_data_to_c!(self, genesis_index, msg, receiveFinalizationMessage)
-        .check_consistent()
+            .check_consistent()
```

Comment:
These changes seem to be necessary to make the rust format check happy, not sure why this has changed.